server/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+reset
+set -e
+RELEASES=~/Releases
+ROOT=~/workspace/RedHat/infinispan/infinispan
```

Comment:
where do these directories come from? Is this script used anywhere?

Reply:
Nope! That's a local file of mine that accidentally got included, removing.